### PR TITLE
🐛 Destination S3: Fix Parquet LZO compression

### PR DESCRIPTION
**Pull request by edgao**
revert https://github.com/airbytehq/airbyte/pull/21085 + add a test case to actually run an lzo compression sync. seen in https://github.com/airbytehq/oncall/issues/2028

the test fails without the revert + passes after the revert.

it's still broken in destination-gcs, but I don't want to install java 8 + lzo libraries there... issue to do something about this https://github.com/airbytehq/airbyte-internal-issues/issues/1774

### CODE REVIEW

**Comment by edgao** [comment]
hm, test is passing but I'm still seeing errors when I try running a sync in oss :/

**Comment by edgao** [comment]
nope, I'm just dumb and built the image from the wrong branch 🤦 @airbytehq/destinations ptal :D

**Review by cynthiaxyin — APPROVED** [review]

File: airbyte-integrations/connectors/destination-s3/src/test-integration/java/io/airbyte/integrations/destination/s3/S3ParquetDestinationAcceptanceTest.java [hunk 1]
```diff
@@ -25,4 +36,23 @@ protected JsonNode getBaseConfigJson() {
     return S3DestinationTestUtils.getBaseConfigJsonFilePath();
   }
 
+  /**
+   * Quick and dirty test to verify that lzo compression works. Probably has some blind spots related to cpu architecture.
+   * <p>
+   * Only verifies that it runs successfully, which is sufficient to catch any issues with installing the lzo libraries.
+   */
+  @Test
+  public void testLzoCompression() throws Exception {
```
Comment (ryankfu) [inline airbyte-integrations/connectors/destination-s3/src/test-integration/java/io/airbyte/integrations/destination/s3/S3ParquetDestinationAcceptanceTest.java:45]: LGTM

**Review by ryankfu — APPROVED** [review]

**Comment by edgao** [comment]
merging without publish because we have publish on merge!

**Pull request merged**